homeassistant/components/zha/config_flow.py

```diff
@@ -242,6 +242,39 @@ async def async_step_port_config(self, user_input=None):
             errors=errors,
         )
 
+    async def async_step_onboarding(self, data=None):
```

Comment:
This step will be initiated by the home assistant yellow hardware integration, should we name the step "hardware" instead?

Reply:
Renamed to "hardware"